tm2py/components/network/highway/highway_network.py

```diff
@@ -76,6 +77,34 @@ def run(self):
                 self._calc_link_class_costs(network)
                 scenario.publish_network(network)
 
+    def validate_inputs(self):
```

Comment:
perhaps a method in the abstract class for "verify_input_file" which does this so we don't have to write the same thing over and over again.

Reply:
...and maybe use pydantic for csvs and json-schema for json files?